src/GovUk.Education.ExploreEducationStatistics.Data.Model/Migrations/20210209094711_EES1836_RemoveObservationRowFilterItemIdentityId.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace GovUk.Education.ExploreEducationStatistics.Data.Model.Migrations
+{
+    public partial class EES1836_RemoveObservationRowFilterItemIdentityId : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.Sql("ALTER TABLE ObservationRowFilterItem DROP CONSTRAINT PK_ObservationRowFilterItem");
+            migrationBuilder.Sql("ALTER TABLE ObservationRowFilterItem DROP COLUMN Id");
+            migrationBuilder.Sql("ALTER TABLE ObservationRowFilterItem ADD CONSTRAINT PK_ObservationRowFilterItem PRIMARY KEY CLUSTERED (ObservationId, FilterItemId)");
+        }
+
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+
```

Comment:
No down migration?

Reply:
Nahhhhh.

No I didn't think it necessary, as these tables aren't currently being used :)  They're just getting ready to go, and if for whatever reason we have a failure here, it'll just be a case of fixing manually anyway, as we don't want this column anymore :)

Reply:
Cool sounds good!